actionpack/test/dispatch/header_test.rb

```diff
@@ -105,16 +105,20 @@ def make_headers(hash)
   end
 
   test "#merge! headers with mutation" do
+    # rubocop:disable Performance/RedundantMerge
```

Comment:
One of the core teams' design goals when adding Rubocop was that we never wanted to add these disable comments. So we've intentionally turned on a light set of cops to give us some enforcement, but not a straitjacket where we'd have to strip the buttons off again.

Let's find another to write this or we just don't enable that cop.

Reply:
Based on my reading here, it appears this is testing external use of `merge!` which we don't have control over. I could rewrite it to use the performance improvement version but it seems to go against the spirit of the test.

Reply:
The only other way would be “send”. I think that’s worse than the comment and it’s only in a handful of places.